package.json

```diff
@@ -10,13 +10,15 @@
   "main": "./index.js",
   "types": "./index.d.ts",
   "dependencies": {
-    "@vue/compiler-sfc": "^3.2.20",
     "svgo": "^3.0.2"
   },
   "devDependencies": {
     "cypress": "^8.6.0",
     "standard": "^16.0.3"
   },
+  "peerDependencies": {
+    "vue": "3.3.3"
```

Comment:
Does this mean you have to be on the exact version 3.3.3 of vue? It would be better if this works for all versions from 3.2.13 and higher (because from that version compiler-sfc is included as a dependency of the main vue package)

Reply:
fixed in https://github.com/jpkleemans/vite-svg-loader/pull/105 :-)